src/main/java/com/google/sps/servlets/ErrorHandlingUtils.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2010 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Utility class for common functionality
+ */
+class ErrorHandlingUtils {
+
+  static void setError(int errorCode, String errorMessage, HttpServletResponse response) throws IOException {
+    response.setContentType("application/text");
+    response.getWriter().println(errorMessage);
+    response.setStatus(errorCode);
+  }
+
+
+  private ErrorHandlingUtils() {
```

Comment:
Did you mean to make it abstract instead so that callers cannot instantiate this class?

Reply:
That might be better but I was just following the example from the Utils.java class